src/Paramore.Brighter/Message.cs

```diff
@@ -76,6 +88,42 @@ public Message(MessageHeader header, MessageBody body)
         {
             Header = header;
             Body = body;
+            
+            UpdateCloudEventsFromHeaders();
+        }
+
+        /// <summary>
+        /// Populate the Cloud Events from the HeaderBag
+        /// </summary>
+        [Obsolete("Looking to remove this in v10 in favour of storing the Cloud Events in their own property in the outbox")]
+        public void UpdateCloudEventsFromHeaders()
```

Comment:
Same theme, there are just header properties for tracing, we can do CE support later

Reply:
Renamed

Reply:
Clearer